app/views/admin/events/index.html.erb

```diff
@@ -8,18 +8,20 @@
     <th></th>
   </tr>
   <% @events.each do |event| %>
-    <tr>
-      <td><%= event.name %></td>
-      <td>
-      <% if event.active %>
-        <span class="badge bg-success">Yes</span>
-      <% else %>
-        <span class="badge bg-danger">No</span>
-      <% end %>
-      </td>
-      <td><%= event.event_mode.to_s.titleize %></td>
-      <td><%= link_to 'Open', admin_event_path(event) %></td>
-    </tr>
+    <% cache event do %>
```

Comment:
can we remove this fragment caching commit? I don't want to introduce too much change right now, and this doesn't feel essential.

Reply:
From the admin side I'm down to remove it, but from the public pages I'd like to keep it in place and as it's powered by Rails by default, I'm not worried about any issues this has